### PR TITLE
[TFL FE] Add CONV3D op support

### PR DESCRIPTION
**Pull request by akodanka**
### Details:
 - Add Conv3D op implementation to TFLite
 - Add testcases for Conv3D

### Tickets:
 - CVS-178650

### AI Assistance:
 - AI assistance used: yes
 - Copilot was used to generate test code, and to fix errors in code
 - Manually reviewed the changes, and verified the test cases by running the tests

### CODE REVIEW

File: tests/layer_tests/tensorflow_lite_tests/test_tfl_Conv3D.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+                                                    name=self.inputs[0])
+            tf.nn.conv3d(place_holder, weights, params['strides'], params['padding'], 'NDHWC',
+                         params['dilations'], name=self.outputs[0])
+            net = sess.graph_def
+        return net
+
+    @pytest.mark.parametrize("params", test_params)
+    @pytest.mark.nightly
+    def test_conv3d(self, params, ie_device, precision, temp_dir):
+        self._test(ie_device, precision, temp_dir, {**params, 'custom_eps': 0.5})
```
[the remark this answers is not in the document]
Comment (akodanka) [reply tests/layer_tests/tensorflow_lite_tests/test_tfl_Conv3D.py:44]: Since the CPU by default runs with bf16 precision, the epsilon value has to be set low for realistic randn values to avoid the accuracy errors. This is true for all convolution testcases too, where randint is being used to reduce the error.

File: tests/layer_tests/tensorflow_lite_tests/test_tfl_Conv3D.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+
+class TestTFLiteConv3DLayerTest(TFLiteLayerTest):
+    inputs = ["Input"]
+    outputs = ["Conv3D"]
+    allowed_ops = ['CONV_3D']
+
+    def make_model(self, params):
+        assert len(set(params.keys()).intersection({'shape', 'ksize', 'strides',
+                                                    'padding', 'dilations'})) == 5, \
```
[the remark this answers is not in the document]
Comment (akodanka) [reply tests/layer_tests/tensorflow_lite_tests/test_tfl_Conv3D.py:29]: The expectation is to make sure all the expected keys are present.

File: src/frontends/tensorflow_lite/src/op/conv3d.cpp [hunk 3]
```diff
@@ -0,0 +1,34 @@
+// Copyright (C) 2018-2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "common_op_table.hpp"
+#include "op_translation_utils.hpp"
+#include "utils.hpp"
+
+using namespace std;
+
```
[the remark this answers is not in the document]
Comment (akodanka) [reply src/frontends/tensorflow_lite/src/op/conv3d.cpp]: Done

**Review by mvafin — APPROVED** [review]
